observability/developer-metrics/serverless-developer-metrics.adoc

```diff
@@ -8,7 +8,7 @@ toc::[]
 
 Metrics enable developers to monitor how Knative services are performing. You can use the {ocp-product-title} monitoring stack to record and view health checks and metrics for your Knative services.
 
-You can view different metrics for {ServerlessProductName} by navigating to link:https://docs.openshift.com/container-platform/latest/observability/monitoring/reviewing-monitoring-dashboards.html#reviewing-monitoring-dashboards-developer_reviewing-monitoring-dashboards[*Dashboards*] in the  web console *Developer* perspective.
+You can view different metrics for {ServerlessProductName} by navigating to link:https://docs.openshift.com/container-platform/latest/observability/monitoring/reviewing-monitoring-dashboards.html#reviewing-monitoring-dashboards-developer_reviewing-monitoring-dashboards[*Dashboards*] in the  web console.
```

Comment:
The formatting for "Dashboards is not needed and like my previous command, the link should point to an existing section.